### ISSUE TITLE
Error from Example

### ISSUE THREAD

**Issue by devYusuf** [denjucks/session#4]
I ran the example made for oak framework from the README.md an got the errors
```sh
thread 'main' panicked at 'already borrowed: BorrowMutError', /rustc/4fb7144ed159f94491249e86d5bbd033b5d60550/src/libcore/cell.rs:878:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
Aborted

```

**Comment by denjucks** [comment]
Looks like its an issue with newer versions of deno. Try running `deno upgrade` and let me know if it persists.

**Comment by denjucks** [comment]
Should be fixed in Deno v1.1.0
